src/Microsoft/Extensions/Silk.NET.Direct3D11.Extensions.D3D11on12/Direct3D11on12.gen.cs

```diff
@@ -0,0 +1,829 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+using System;
+using System.Runtime.InteropServices;
+using System.Runtime.CompilerServices;
+using System.Text;
+using Silk.NET.Core;
+using Silk.NET.Core.Native;
+using Silk.NET.Core.Attributes;
+using Silk.NET.Core.Contexts;
+using Silk.NET.Core.Loader;
+using Silk.NET.Direct3D11;
+using Extension = Silk.NET.Core.Attributes.ExtensionAttribute;
+
+#pragma warning disable 1591
+
+namespace Silk.NET.Direct3D11.Extensions.D3D11on12
+{
+    [Extension("d3d11on12.h")]
+    public unsafe partial class Direct3D11on12 : NativeExtension<D3D11>
```

Comment:
This should probably be `D3D11On12` to keep consistent with the main class being `D3D11` as well as to keep consistent with pascal case